features/standard/ContentTranslation.feature

```diff
@@ -0,0 +1,44 @@
+@javascript @translation @IbexaOSS @IbexaContent @IbexaExperience @IbexaCommerce
+Feature: Content item transation
+
+  @javascript @APIUser:admin
+  Scenario: Publish new translation based on existing translation
+    Given I create "folder" Content items in root in "eng-GB"
+      | name             | short_name       | short_description | description      |
+      | EnglishPublished | EnglishPublished | EnglishPublished  | EnglishPublished |
+    And I am logged as admin
+    And I'm on Content view Page for "EnglishPublished"
+    When I switch to "Translations" tab in Content structure
+    And I add new translation "French" basing on "English (United Kingdom)" translation
+    And I set content fields
+      | label      | value           |
+      | Name       | FrenchPublished |
+      | Short name | FrenchPublished |
+    And I click on the edit action bar button "Publish"
+    Then success notification that "Content published." appears
```

Comment:
What do you think about checking the English preview here as well? Just to make sure that it doesn't somehow change?
```suggestion
    Then success notification that "Content published." appears
    And content attributes equal
      | label             | value            |
      | Name              | EnglishPublished |
      | Short name        | EnglishPublished |
      | Short description | EnglishPublished |
      | Description       | EnglishPublished |
```

Reply:
Sure, I can try adding that.